### ISSUE TITLE
how to setting the number of  train iteration about rpn and fast rcnn

### ISSUE THREAD

**Issue by alexchung16** [DetectionTeamUCAS/Faster-RCNN_Tensorflow#35]
when i running the train,py , the terminal only print the rpn_cla_loss infomation, another loss that
rpn_loc_loss,fast_rcnn_loc_loss,fast_rcnn_cla_loss and so on seem not execute train.
as the paper elaborate of faster rcnn that the training step should contain four step.
